hw/top_earlgrey/dv/verilator_sim_cfg.hjson

```diff
@@ -108,14 +140,14 @@
     {
       name: dif_otbn_smoketest_rtl
       sw_images: ["sw/device/tests/dif_otbn_smoketest:1"]
-      run_opts: ['+OTBN_USE_MODEL=0']
+      run_opts: ["+OTBN_USE_MODEL=0"]
     }
-    // Using the model in CI isn't possible until #4097 is resolved.
-    //{
-    //  name: dif_otbn_smoketest_model
-    //  sw_images: ["sw/device/tests/dif_otbn_smoketest:1"]
-    //  run_opts: ['+OTBN_USE_MODEL=0']
-    //}
+    // TODO: Needs to be debugged.
```

Comment:
nit: I think it would be nice to have a test_disable knob so that we don't have commented out config code.

Reply:
Ideally I would have left this uncommented (let the test fail so that we debug it). But unfortunately the test hangs indefinitely, and the verilator testbench does not support a timeout (AFAIK). So the required fix is to add a programmable sim-time timeout within `chip_earlgrey_verilator` to gracefully kill the test. Of course, DVSim also needs to support wall-clock timeout.  Filed #6659 and #6661 respectively. 

Also, we don't run this in CI. This is an alternative to the `pytest` way of running Verilator tests. As much as I would like everyone to use it, it does not seem to have gained the traction I was hoping for. 

DVSim already supports the concept of running different types of regressions (running collections of tests). There is one for CI (run a subset of tests for quickness) and one for nightly (run everything). I have not enabled those yet.